cubeviz/conftest.py

```diff
@@ -65,7 +65,7 @@
 from .tests.helpers import (toggle_viewer, select_viewer, create_glue_app,
                             reset_app_state)
 
-@pytest.fixture(scope='session')
+@pytest.fixture(scope='module')
```

Comment:
Is this going to be necessary in the long term or was it just an experiment to try to make the tests pass?

Reply:
I'm not quite sure how to tackle this issue: locally, it seems that a session-scoped fixture with the `reset_state` function is not enough to clear the state of the gui. The only way I could get the tests to pass locally was if the applications was hard reset before every module.

Reply:
I think we'll have to take a look at the `reset_app_state` in greater detail. But the tests were not significantly longer when scoped with `module`.